### PR TITLE
docs(kit): Sketch grid content

### PR DESCRIPTION
**Pull request by aagonzales**
Updating the content around the Sketch grid in order to better help designers understand how to use the grid function in Sketch.

### CODE REVIEW

**Review by tw15egan — APPROVED** [review]

**Review by aledavila — APPROVED** [review]

File: src/pages/designing/kits/sketch.mdx [hunk 1]
```diff
@@ -172,26 +165,63 @@ page for an overview of the kit.
...
+
+An influencer is a component that affects the content on the page. It can either appears on a page as the result of a user action or be part of your product's page. These [influencers](/guidelines/2x-grid/overview/#grid-influencers) effect the layout grid by scaling and resizing the columns and its content. You can find an assortment of examples of grids with an influences in the grid template file on the page labeled "Grid influencers." 
+
+### Calculating the Sketch grid
+
+#### Sketch layout settings
+
+Slide-in panels influence the page layout grid; below are some of the specifications for panel combinations and how they impact the grid at all sizes.
+
+- Breakpoint Width* = **Artboard Width**
```
Comment (janchild) [inline src/pages/designing/kits/sketch.mdx:206]:
```suggestion
- Breakpoint width* = **Artboard width**
```

File: src/pages/designing/kits/sketch.mdx [hunk 2]
```diff
@@ -172,26 +165,63 @@ page for an overview of the kit.
...
+- Breakpoint Gutter* = **Left and Right Outside Margins**
```
Comment (janchild) [inline src/pages/designing/kits/sketch.mdx:207]:
```suggestion
- Breakpoint gutter* = **Left and right outside margins**
```

File: src/pages/designing/kits/sketch.mdx [hunk 3]
```diff
@@ -172,26 +165,63 @@ page for an overview of the kit.
...
+- Total Panel Width (“sidebar nav”) + Left outside margin + Right outside margin = **Total Margins**
```
Comment (janchild) [inline src/pages/designing/kits/sketch.mdx:208]:
```suggestion
- Total panel width (“sidebar nav”) + Left outside margin + Right outside margin = **Total margins**
```

File: src/pages/designing/kits/sketch.mdx [hunk 4]
```diff
@@ -172,26 +165,63 @@ page for an overview of the kit.
...
+- Artboard Width - Total Margins = **Total Width**
```
Comment (janchild) [inline src/pages/designing/kits/sketch.mdx:209]:
```suggestion
- Artboard width - Total margins = **Total width**
```

File: src/pages/designing/kits/sketch.mdx [hunk 5]
```diff
@@ -172,26 +165,63 @@ page for an overview of the kit.
...
+- Total Panel Width + Left margin = **Offset**
```
Comment (janchild) [inline src/pages/designing/kits/sketch.mdx:210]:
```suggestion
- Total panel width + Left margin = **Offset**
```

File: src/pages/designing/kits/sketch.mdx [hunk 6]
```diff
@@ -172,26 +165,63 @@ page for an overview of the kit.
...
+
+*Since we are following the Carbon Design responsive guidelines, we referenced [this table](https://www.carbondesignsystem.com/guidelines/2x-grid/implementation#responsive-options) to determine our common breakpoint widths and respective gutter specs.
+
+#### Example
+
+Let’s say we want to create a custom grid for a new design that uses the extra small panel (256px wide) on our x-large 1312px breakpoint. Applying the formula above, your calculations would look like this
 
-The grid template includes templates for all breakpoints along with the
-necessary formulas to design for custom screen sizes.
+- Breakpoint Width (1312px) = 1312px wide artboard
```
Comment (janchild) [inline src/pages/designing/kits/sketch.mdx:218]:
```suggestion
- Breakpoint width (1312px) = 1312px wide artboard
```

File: src/pages/designing/kits/sketch.mdx [hunk 7]
```diff
@@ -172,26 +165,63 @@ page for an overview of the kit.
...
+- Breakpoint gutter (32px) = 32px Left outside margin and 32px Right outside margin
+- Total Panel Width (256px) + 16px Left + 16px Right = 288px Total Margins
```
Comment (janchild) [inline src/pages/designing/kits/sketch.mdx:220]:
```suggestion
- Total panel width (256px) + 16px Left + 16px Right = 288px Total margins
```

File: src/pages/designing/kits/sketch.mdx [hunk 8]
```diff
@@ -172,26 +165,63 @@ page for an overview of the kit.
...
+- Artboard Width (1312px) - Total Margins (272px) = **1,056px Total Width**
```
Comment (janchild) [inline src/pages/designing/kits/sketch.mdx:221]:
```suggestion
- Artboard width (1312px) - Total margins (272px) = **1,056px Total width**
```

File: src/pages/designing/kits/sketch.mdx [hunk 9]
```diff
@@ -172,26 +165,63 @@ page for an overview of the kit.
...
+- Total Panel Width (256px) + 16px Left = **272px Offset**
```
Comment (janchild) [inline src/pages/designing/kits/sketch.mdx:222]:
```suggestion
- Total panel width (256px) + 16px Left = **272px Offset**
```

**Review by janchild — COMMENTED** [review]
Hi @aagonzales, this looks great! Just a few little copyedit tweaks.

**Pull request merged**